docs/modules/ROOT/pages/explanations/jsonnet.adoc

```diff
@@ -110,3 +110,50 @@ This is to allow adding extra config without redefining the full default config.
 ////
 * One `main.jsonnet` and one `compile` instruction
 ////
+
+== Randomize (cron) schedules
+
+Randomize CronJob or any other time based schedules is a crucial topic.
+
+With Kapitan and Jsonnet, we're limited to the scope of a single cluster when compiling a catalog.
+Distributing schedules must be done within that limited scope.
+This can be done by leveraging a hashing function.
+The input to that hashing function then defines the scope of distribution.
+
+[source,jsonnet]
+----
+local scope = "…" // Something that defines the scope of the schedule
+local minute = std.foldl(function(x, y) x + y, std.encodeUTF8(std.md5(scope)), 0) % 60;
+local job = kube.CronJob(name) {
+  spec+: {
+    schedule: '%d * * * *' % minute,
+  …
+}
+----
+
+There are three obvious cases to distribute schedules:
+
+* Distributed clients, single target:
+  Clients are distributed through multiple clusters but act against a single API target.
+  Within the scope of all clusters, each client should take action at a different time.
+  This is to not overload the API.
++
+Use `inv.parameters.cluster.name` as the scope.
+
+* Workload on same cluster:
+  Multiple job instances are scheduled on the same cluster.
+  Within the scope of a single clusters, each job needs to start at a different time.
+  This is to not exhaust a clusters available resources.
```

Comment:
```suggestion
  This is to not exhaust a clusters available resources and to avoid increased scheduling times due to many jobs needing to be scheduled in a short time.
```